### ISSUE TITLE
Relation Modal Popup - "Namespace" Placeholder

### ISSUE THREAD

**Issue by lloy0076** [thedevdojo/voyager#1908]
- Laravel Version: 5.5.X
- Voyager Version: 1.0.4
- PHP Version: 7.1
- Database Driver & Version: N/A

### Description:

Relation creating model implies that it needs **just** the ` Namespace` but it actually needs the full model name.

I was setting up a ManyToMany but this may apply to other types too.

So, I put in:

* `App\`

Instead of...

* `App\AuthorBook`

### Steps To Reproduce:

Setup voyager with two relatable BREADs and a pivot table (although this probably applies to other types too). Then create a relationship and put *just* `App\` in the field with the current `Namespace` holder (in grey by default).

**Comment by fletch3555** [comment]
I agree, `Namespace` isn't specific enough for this.  Do you have any suggestions?

**Comment by lloy0076** [comment]
http://php.net/manual/en/language.namespaces.rules.php

```
Fully qualified name

This is an identifier with a namespace separator that begins with a namespace
separator, such as \Foo\Bar. The namespace \Foo is also a fully qualified name.
```

`Fully Qualified Name` or `Fully Qualified Class Name` might be good and if it's translatable you could probably look at the manual's various translations for the foreign word equivalent(s).

**Comment by alexisferat** [comment]
+1 it is very confusing for newcomers. I spent 10 minutes debugging until I realized that it require the model FQDN.

**Comment by emptynick** [comment]
Fixed in #3909 commit https://github.com/the-control-group/voyager/pull/3909/commits/72adc7fe421b729430ba689e91f6094428bc960b
